components/xwiki-platform-users/xwiki-platform-users-wiki/src/main/java/org/xwiki/users/internal/WikiUser.java

```diff
@@ -94,10 +94,10 @@ public String getName()
         if (this.profileReference == null) {
             return "";
         }
-        String result = this.bridge.getProperty(this.profileReference, this.classReference, "first_name") + "";
+        String result = this.bridge.getProperty(this.profileReference, this.classReference, "first_name");
```

Comment:
This causes compilation failure, the `+""` was used to convert it to a string. `String.valueOf` would be better.